src/gsumm.c

```diff
@@ -896,39 +930,62 @@ SEXP gmax(SEXP x, SEXP narm)
     break;
   case REALSXP:
```

Comment:
There's a `TODO` above:

> // TODO rework gmax in the same way as gmin and remove this *update

I _partially_ addressed that in this PR, only for the `REALSXP` case